src/NHPersistenceCompatibilityTests/4.5/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Assembly info is metadata for the purpose of deployment. So i assume none of them are required in this sln